src/Admin/BlockAdmin.php

```diff
@@ -205,7 +230,7 @@ public function setContainerBlockTypes(array $containerBlockTypes): void
     }
 
     /**
-     * @param $defaultContainerType
+     * @param string $defaultContainerType
```

Comment:
can you just remove this comment and add `string` typehint as you are targeting master and we have `7.1` there